app/helpers/vacancies_helper.rb

```diff
@@ -199,6 +199,10 @@ def vacancy_publication_status(vacancy)
     end
   end
 
+  def section_to_first_step(section, step_process)
+    step_process.step_groups.fetch(section).first
+  end
```

Comment:
Suggestion: 
Since this logic seems only to be concerned with retrieving the first step for the given step group,  wouldn't it make sense to define it within the  `StepProcess` class as `first_step_for_group` or `step_group_first_step` or ... ?

And then could be called over `@step_process.first_step_for_group(section)`